### DISCUSSION TITLE
[FR]: status bar for pagination

### DISCUSSION THREAD

**Discussion by imed-ghomari** [RafaelGB/obsidian-db-folder#691] [Ideas]
### Present your request

First, I recommend showing information for only the active database file, because showing the three status bar items for three databases open is confusing for users.

Second, for pagination, I recommend : 
1. adding the`«` `<` `>` `»` 
2. removing the name of the database because it only works with current open databases (as mentioned above)
3. adding the number of all rows + the number of the current page + the number of all the pages

An example would be like this : `« < > » (2/3) 36` where 36 is the total number of rows, 2 is the current page and 3 is the total number of pages.

![Screen Shot 2022-12-23 at 10 40 58 PM](https://user-images.githubusercontent.com/38974541/209408434-b211071c-3333-49cb-a83b-b050102a36d0.png)

The advantages of this method are :
- You won't have to adapt last row because of pagination buttons
- It'll feel more native to obsidian
- You have more space, so you can display more rows.

### For which platform do you request this request??

Cross

**Comment by RafaelGB** [comment]
First and as always, thank you for your very detailed idea.  

In 3.0.0 I tried this approach but technically it has its drawbacks apart from the fact that it would not support mobile phones since they do not have said bar.

About the information of the ddbbs just affects those files opened leaving the fist place for the active one. Maybe we can change the color for the active table

**Comment by imed-ghomari** [comment]
I see, yeah, didn't think about mobile, you're right.

**Comment by RafaelGB** [comment]
we can move this to ideas discussed, the pagination could be improved for sure :)
